custom_components/opnsense/coordinator.py

```
@@ -70,162 +92,180 @@ async def _get_states(self, categories: list) -> MutableMapping[str, Any]:
+        if self._state.get("device_unique_id") is None:
+            _LOGGER.warning("Coordinator failed to confirm OPNsense Router Unique ID. Will retry")
+            self._mismatched_count = 0
+            return False
+        if self._state.get("device_unique_id") != self._device_unique_id:
+            _LOGGER.debug(
+                "[Coordinator async_update_data]: config device id: %s, router device id: %s",
+                self._device_unique_id,
+                self._state.get("device_unique_id"),
+            )
+            _LOGGER.error(
+                "Coordinator error. OPNsense Router Device ID (%s) differs from the one saved in hass-opnsense (%s)",
```

Pylint (3.3.7)

[convention] Line too long (117/100)

(C0301)